src/maconn.ts

```diff
@@ -64,6 +64,12 @@ export class WebRTCMultiaddrConnection implements MultiaddrConnection {
     this.remoteAddr = init.remoteAddr
     this.timeline = init.timeline
     this.peerConnection = init.peerConnection
+
+    this.peerConnection.onconnectionstatechange = () => {
```

Comment:
Not sure if this is necessary, we have [an event listener](https://github.com/libp2p/js-libp2p-webrtc/blob/4f1840e04da926e49ae24f5f7639780a1aef4155/src/transport.ts#L216-L217) that will close these connections in the transport as well as in the [peer transport](https://github.com/libp2p/js-libp2p-webrtc/blob/4f1840e04da926e49ae24f5f7639780a1aef4155/src/peer_transport/handler.ts#L43)

Reply:
We need something else - that event listener only handles outbound connections.  We need to handle inbound connections too.